upgrades/steps_28_test.go

```diff
@@ -316,7 +316,7 @@ func assertSetUnitStateArg(c *gc.C, expectedArg, obtainedArg params.SetUnitState
 	if expectedArg.RelationState != nil {
 		c.Assert(*obtainedArg.RelationState, gc.DeepEquals, *expectedArg.RelationState)
 	} else {
-		c.Assert(obtainedArg.State, gc.IsNil)
+		c.Assert(obtainedArg.RelationState, gc.IsNil)
```

Comment:
This one was a typo and has now been fixed.

Reply:
👍